diff --git a/node_modules/simple-qrbtf/build/main/lib/component/base.qr.js b/node_modules/simple-qrbtf/build/main/lib/component/base.qr.js
index 60168ba..9891f16 100644
--- a/node_modules/simple-qrbtf/build/main/lib/component/base.qr.js
+++ b/node_modules/simple-qrbtf/build/main/lib/component/base.qr.js
@@ -15,6 +15,7 @@ var PosType;
     PosType["Round"] = "round";
     PosType["Planet"] = "planet";
     PosType["RoundRect"] = "roundRect";
+    PosType["CustomStyle1"] = "customstyle1";
 })(PosType || (PosType = {}));
 const BaseQrRenderer = {
     defaultProps: {
@@ -38,6 +39,7 @@ const BaseQrRenderer = {
         const vw = [3, -3];
         const vh = [3, -3];
         const sq25 = 'M32.048565,-1.29480038e-15 L67.951435,1.29480038e-15 C79.0954192,-7.52316311e-16 83.1364972,1.16032014 87.2105713,3.3391588 C91.2846454,5.51799746 94.4820025,8.71535463 96.6608412,12.7894287 C98.8396799,16.8635028 100,20.9045808 100,32.048565 L100,67.951435 C100,79.0954192 98.8396799,83.1364972 96.6608412,87.2105713 C94.4820025,91.2846454 91.2846454,94.4820025 87.2105713,96.6608412 C83.1364972,98.8396799 79.0954192,100 67.951435,100 L32.048565,100 C20.9045808,100 16.8635028,98.8396799 12.7894287,96.6608412 C8.71535463,94.4820025 5.51799746,91.2846454 3.3391588,87.2105713 C1.16032014,83.1364972 5.01544207e-16,79.0954192 -8.63200256e-16,67.951435 L8.63200256e-16,32.048565 C-5.01544207e-16,20.9045808 1.16032014,16.8635028 3.3391588,12.7894287 C5.51799746,8.71535463 8.71535463,5.51799746 12.7894287,3.3391588 C16.8635028,1.16032014 20.9045808,7.52316311e-16 32.048565,-1.29480038e-15 Z';
+        const sq26 = 'M18.6 3H5.4A2.4 2.4 0 0 0 3 5.4v13.2A2.4 2.4 0 0 0 5.4 21h13.2a2.4 2.4 0 0 0 2.4-2.4V5.4A2.4 2.4 0 0 0 18.6 3Z';
         if (size <= 0)
             size = 1.0;
         for (let x = 0; x < nCount; x++) {
@@ -61,7 +63,7 @@ const BaseQrRenderer = {
                 else if (typeTable[x][y] === utils_1.QRPointType.POS_CENTER) {
                     if (posType === PosType.Rect) {
                         pointList.push(`
-              <rect width='${1}' height='${1}' fill='${posColor}' x='${x}' y='${y}' />
+              <rect width='${1.05}' height='${1.05}' fill='${posColor}' x='${x}' y='${y}' />
           `);
                     }
                     else if (posType === PosType.Round) {
@@ -86,16 +88,43 @@ const BaseQrRenderer = {
                     }
                     else if (posType === PosType.RoundRect) {
                         pointList.push(`
-              <circle fill='${posColor}' cx='${x + 0.5}' cy='${y + 0.5}' r='${1.5}' /> `);
+                          <circle fill='${posColor}' cx='${x + 0.5}' cy='${
+                          y + 0.5
+                        }' r='${1.5}' /> `);
                         pointList.push(`
-              <path d='${sq25}' stroke='${posColor}' stroke-width='${(100 / 6) * (1 - (1 - size) * 0.75)}' fill='none' transform='${'translate(' +
-                            String(x - 2.5) +
+                          <path d='${sq25}' stroke='${posColor}' stroke-width='${
+                          (100 / 6) * (1 - (1 - size) * 0.75)
+                        }' fill='none' transform='${
+                          'translate(' +
+                          String(x - 2.5) +
+                          ',' +
+                          String(y - 2.5) +
+                          ') scale(' +
+                          String(6 / 100) +
+                          ',' +
+                          String(6 / 100) +
+                          ')'
+                        }' />
+                      `);
+                    }
+                    else if (posType === PosType.CustomStyle1) {
+                        pointList.push(`
+                        <rect width='${2.8}' height='${2.8}' fill='${posColor}' x='${x - 0.9}' y='${y - 0.9}' rx='${0.5}' /> `);
+                        pointList.push(`
+                        <rect width='${0.8}' height='${0.8}' fill='${posColor}' x='${x - 0.88}' y='${y + 1.09}' /> `);
+                        pointList.push(`
+                        <rect width='${1.15}' height='${1.2}' fill='${posColor}' x='${x - 3.05}' y='${y + 2.88}' /> `);
+                        pointList.push(`
+              <path d='${sq26}' stroke='${posColor}' stroke-width='${
+                (26 / 6) * (1 - (1 - size) * 0.75)
+              }' fill='none' transform='${'translate(' +
+                            String(x - 3.5) +
                             ',' +
-                            String(y - 2.5) +
+                            String(y - 3.4) +
                             ') scale(' +
-                            String(6 / 100) +
+                            String(33 / 100) +
                             ',' +
-                            String(6 / 100) +
+                            String(33 / 100) +
                             ')'}' />
           `);
                     }
@@ -103,7 +132,7 @@ const BaseQrRenderer = {
                 else if (typeTable[x][y] === utils_1.QRPointType.POS_OTHER) {
                     if (posType === PosType.Rect) {
                         pointList.push(`
-              <rect width='${1}' height='${1}' fill='${posColor}' x='${x}' y='${y}' />
+              <rect width='${1.05}' height='${1.05}' fill='${posColor}' x='${x}' y='${y}' />
           `);
                     }
                 }
diff --git a/node_modules/simple-qrbtf/build/main/lib/renderer.js b/node_modules/simple-qrbtf/build/main/lib/renderer.js
index 386686a..0204db3 100644
--- a/node_modules/simple-qrbtf/build/main/lib/renderer.js
+++ b/node_modules/simple-qrbtf/build/main/lib/renderer.js
@@ -10,13 +10,13 @@ function defaultViewBox({ qrcode, }) {
     if (!qrcode)
         return '0 0 0 0';
     const nCount = qrcode.getModuleCount();
-    return (String(-nCount / 5) +
+    return '0' +
         ' ' +
-        String(-nCount / 5) +
+        '0' +
         ' ' +
-        String(nCount + (nCount / 5) * 2) +
+        String(nCount) +
         ' ' +
-        String(nCount + (nCount / 5) * 2));
+        String(nCount);
 }
 exports.defaultViewBox = defaultViewBox;
 function defaultDrawIcon({ qrcode, icon, }) {
@@ -82,11 +82,32 @@ function getDefaultIcon(iconMode) {
   <path d='M41.055,57.675a2.183,2.183,0,0,1-2.893-.883l-.143-.314L32.046,43.37a1.133,1.133,0,0,1-.105-.461,1.094,1.094,0,0,1,1.748-.877l7.049,5.019a3.249,3.249,0,0,0,2.914.333L76.8,32.63c-5.942-7-15.728-11.581-26.8-11.581-18.122,0-32.813,12.243-32.813,27.345,0,8.24,4.42,15.656,11.338,20.669a2.185,2.185,0,0,1,.919,1.781,2.569,2.569,0,0,1-.116.7c-.552,2.062-1.437,5.362-1.478,5.516a3.212,3.212,0,0,0-.177.8,1.094,1.094,0,0,0,1.1,1.094,1.236,1.236,0,0,0,.631-.2L36.583,74.6a3.438,3.438,0,0,1,1.742-.5,3.281,3.281,0,0,1,.965.145A38.844,38.844,0,0,0,50,75.739c18.122,0,32.813-12.243,32.813-27.345a23.668,23.668,0,0,0-3.738-12.671L41.3,57.537Z' fill='#fff'/>
 </g>
     `;
-    const AlipayIcon = `
-<g>
-  <rect width='100' height='100' fill='#009ce1'/>
-  <path d='M100,67.856c-.761-.1-4.8-.8-17.574-5.066-4.012-1.339-9.4-3.389-15.395-5.552A80.552,80.552,0,0,0,75.4,36.156H55.633v-7.1H79.848V25.094H55.633V13.258H45.749a1.68,1.68,0,0,0-1.733,1.707V25.094H19.524v3.963H44.016v7.1H23.8V40.12H63.013a69.579,69.579,0,0,1-5.65,13.763c-12.724-4.187-26.3-7.58-34.834-5.491C17.074,49.733,13.56,52.125,11.5,54.63,2.02,66.125,8.815,83.585,28.824,83.585c11.831,0,23.228-6.579,32.061-17.417C73.49,72.211,97.914,82.4,100,83.267ZM26.956,76.9c-15.6,0-20.215-12.255-12.5-18.958,2.573-2.266,7.276-3.372,9.782-3.621,9.268-.913,17.846,2.613,27.972,7.541C45.087,71.118,36.023,76.9,26.956,76.9Z' fill='#fff'/>
-</g>
+    const SuperrPetsIcon = `
+    <g>
+    <path fill-rule="evenodd" clip-rule="evenodd" fill="#353497" d="M150.9,68.2c-5.7,3.6-8,8.1-8.1,16.5c-0.1,9.8,5.8,19.9,14,23.8c9,4.2,17,0,20.1-10.6c1.3-4.5,1.3-5.3,0.1-10.9
+	c-1.7-8.1-6-14.5-11.8-17.9C160.1,66.1,154.7,65.8,150.9,68.2 M101.5,85.1c-10.8,1.3-15.6,17-9.3,29.9c6.7,13.6,19.8,17.6,27.7,8.5
+	c9.7-11.4,2.7-33-12.4-38.2C106.5,84.9,103.7,84.8,101.5,85.1 M200.9,98.7c-14.4,7.6-13.9,35.2,0.7,38.1c4.4,0.9,8.4-0.4,12.2-4
+	c8.4-7.7,10-22,3.6-31.2C213.9,96.6,207.1,95.4,200.9,98.7 M148.4,128.2c-13.4,3.5-22.1,11.9-25.6,24.9c-0.5,1.8-1,7-1.1,11.6
+	c-0.2,8.2-0.3,8.5-4.5,16.6c-2.3,4.5-4.7,10.2-5.3,12.6c-1.7,7.1,0,16.4,4.3,23.6c9.8,16.4,32.7,21,48.5,9.9
+	c7.8-5.5,15-8.1,22.3-8.1c6.2,0,14.5-2.6,19.9-6.2c11-7.3,16.9-21.9,14-34.4c-2.8-11.7-8.3-18.2-20.5-24.1
+	c-9.9-4.7-12.4-6.7-17.8-13.6C174.5,130.5,160.2,125.1,148.4,128.2 M78.6,140.5c-4.1,1.2-7.6,5.3-8.3,9.5
+	c-1.5,10.1,6.5,20.8,18.5,24.6c4,1.3,4.7,1.3,9.1,0c6.1-1.8,9.2-5.8,9.2-12.1C107.1,149.2,90.9,136.8,78.6,140.5"/>
+    <path fill-rule="evenodd" clip-rule="evenodd" fill="#FACB36" d="M132,28.6c-22.2,3.6-39,9.8-54.7,20.5c-15,10.2-32.2,29.6-40,45c-13.8,27.5-17.2,57.9-9.6,86.7
+	c5.7,21.7,15.9,39.2,32.1,55.4c18.2,18.2,38.1,28.8,63.8,34.1c11.2,2.3,35.5,2.1,46.6-0.4c25.2-5.7,43.9-15.9,61.7-33.7
+	c13.6-13.6,23.1-28.4,29.3-45.8c2.1-6,5.9-22.4,5.9-25.7c0-1.3,0.4-2.6,0.8-2.9c0.5-0.3,0.8-5.5,0.8-12.8c0-6.7-0.3-12-0.6-11.7
+	c-0.3,0.3-1.2-3.3-2-8.1c-1.8-10.4-6.3-24.4-11-33.7c-11.1-22.1-32.5-43.6-54.1-54.5c-10-5-24.2-9.8-32.5-11
+	c-1.8-0.3-5.4-0.8-7.9-1.3C156.2,28,136.4,27.9,132,28.6 M150.9,68.2c-5.7,3.6-8,8.1-8.1,16.5c-0.1,9.8,5.8,19.9,14,23.8
+	c9,4.2,17,0,20.1-10.6c1.3-4.5,1.3-5.3,0.1-10.9c-1.7-8.1-6-14.5-11.8-17.9C160.1,66.1,154.7,65.8,150.9,68.2 M101.5,85.1
+	c-10.8,1.3-15.6,17-9.3,29.9c6.7,13.6,19.8,17.6,27.7,8.5c9.7-11.4,2.7-33-12.4-38.2C106.5,84.9,103.7,84.8,101.5,85.1 M200.9,98.7
+	c-14.4,7.6-13.9,35.2,0.7,38.1c4.4,0.9,8.4-0.4,12.2-4c8.4-7.7,10-22,3.6-31.2C213.9,96.6,207.1,95.4,200.9,98.7 M148.4,128.2
+	c-13.4,3.5-22.1,11.9-25.6,24.9c-0.5,1.8-1,7-1.1,11.6c-0.2,8.2-0.3,8.5-4.5,16.6c-2.3,4.5-4.7,10.2-5.3,12.6
+	c-1.7,7.1,0,16.4,4.3,23.6c9.8,16.4,32.7,21,48.5,9.9c7.8-5.5,15-8.1,22.3-8.1c6.2,0,14.5-2.6,19.9-6.2c11-7.3,16.9-21.9,14-34.4
+	c-2.8-11.7-8.3-18.2-20.5-24.1c-9.9-4.7-12.4-6.7-17.8-13.6C174.5,130.5,160.2,125.1,148.4,128.2 M24.1,150.1c0,6.9,0.1,9.6,0.3,6
+	c0.2-3.6,0.2-9.3,0-12.6C24.3,140.2,24.1,143.2,24.1,150.1 M78.6,140.5c-4.1,1.2-7.6,5.3-8.3,9.5c-1.5,10.1,6.5,20.8,18.5,24.6
+	c4,1.3,4.7,1.3,9.1,0c6.1-1.8,9.2-5.8,9.2-12.1C107.1,149.2,90.9,136.8,78.6,140.5"/>
+    <path fill="#FFFFFF" d="M150,295c-79.9,0-145-65-145-145S70.1,5,150,5s145,65,145,145S229.9,295,150,295z M150,35
+	C86.6,35,35,86.6,35,150s51.6,115,115,115s115-51.6,115-115S213.4,35,150,35z"/>
+    </g>
     `;
     if (iconMode === 2) {
         return WeChatIconSmall;
@@ -98,7 +119,7 @@ function getDefaultIcon(iconMode) {
         return WeChatPayIcon;
     }
     else if (iconMode === 5) {
-        return AlipayIcon;
+        return SuperrPetsIcon;
     }
     return '';
 }
@@ -113,7 +134,7 @@ function builtinDrawIcon({ qrcode, icon, }) {
     if (icon) {
         const iconMode = utils_1.getExactValue(icon.enabled, 0);
         const { scale } = icon;
-        const iconSize = Number((nCount * (scale > 33 ? 33 : scale)) / 100);
+        const iconSize = Number((nCount * (scale > 33 ? 33 : scale)) / 80);
         const iconXY = (nCount - iconSize) / 2;
         if (icon && iconMode) {
             const randomIdDefs = utils_1.getIdNum();
@@ -129,7 +150,7 @@ function builtinDrawIcon({ qrcode, icon, }) {
                 ',' +
                 String(iconSize / 100) +
                 ')';
-            pointList.push(`<path d='${sq25}' stroke='#FFF' stroke-width='${100 / iconSize}' fill='#FFF' transform='${transform}' />`);
+            pointList.push(`<path d='${sq25}' stroke='transparent' stroke-width='${100 / iconSize}' fill='transparent' transform='${transform}' />`);
             const defsTransform = 'translate(' +
                 String(iconXY) +
                 ',' +
@@ -146,9 +167,9 @@ function builtinDrawIcon({ qrcode, icon, }) {
                 String(iconXY) +
                 ') ' +
                 'scale(' +
-                String(iconSize / 100) +
+                String(iconSize / 300) +
                 ',' +
-                String(iconSize / 100) +
+                String(iconSize / 300) +
                 ')';
             pointList.push(`
 <g>
